src/Domain/Brand.php

```diff
@@ -0,0 +1,149 @@
+<?php declare(strict_types = 1);
+
+namespace SandwaveIo\RealtimeRegister\Domain;
+
+use Carbon\Carbon;
+use SandwaveIo\RealtimeRegister\Domain\Enum\LocaleEnum;
+
+final class Brand implements DomainObjectInterface
+{
+    /** @var string */
+    public $customer;
+
+    /** @var string */
+    public $handle;
+
+    /** @var string */
+    public $locale;
+
+    /** @var string */
+    public $organization;
+
+    /** @var string[]|null */
+    public $addressLine;
```

Comment:
This is actually always a `string[]`

Reply:
Good catch :) Fixed here, along with `$abuseContact` field: https://github.com/sandwave-io/realtimeregister-php/pull/25/commits/19c3c70c3531366c59cb0ab1cf93466d2cb0758f